js/popups.js

```diff
@@ -825,9 +830,63 @@ function closeMapClusterPopup(clearMap) {
 function closeAllPopups() {
     deselectLocation();
     deselectAircraft();
+    //closeVoiceMessagePopup();
     closeMapClusterPopup(true);
 }
 
 $(document).ready(function () {
     //window.scrollTo(0,document.body.scrollHeight);
 });
+
+function closeGotoVoiceMessagePopup() {
+    document.getElementById("gottoVoiceMessagePopup").style.display = "none";
+}
+
+function showAudioMessagePopup() {
+    document.getElementById("gottoVoiceMessagePopup").style.display = "none";
+    document.getElementById("myModal").style.display = "flex";
+    playAudioMessageAndTracker()
+}
+
+function closeVoiceMessagePopup(){
+    document.getElementById("audioSRC").pause();
+    document.getElementById("audioSRC").currentTime = 0;
+    document.getElementById("myModal").style.display = "none";
+    document.getElementById("gottoVoiceMessagePopup").style.display = "none";
+}
+
+var audioPLay;
+function playAudioMessageAndTracker() {
+    if(document.getElementById("audioSRC").src !== "")
+    {
+        document.getElementById("audioSRC").play();
+        audioPLay = true;
+    }
+}
+
+function playBTN() {
+    if(document.getElementById("audioSRC").src !== "")
+    {
+        if(audioPLay){
+            audioPLay = false;
+            document.getElementById("audioSRC").pause();
+        }
+        else {
+            audioPLay = true;
+            document.getElementById("audioSRC").play();
+        }
+    }
+}
+
+function updateAudioMessageTime() {
+    activeVoiceMessage = document.getElementById("audioSRC");
```

Comment:
use jquery instead of document.getElementById

Reply:
Done